### DISCUSSION TITLE
I want source to learn Laravel ?

### DISCUSSION THREAD

**Discussion by musaabdullah** [community/community#75942] [Other Feature Feedback, Questions, & Ideas]
I need good source to learn laravel

**Comment by marcusvinicius0** [comment]
Hey there! I've been searching some interesting content and I found out this intire free course: 
(https://www.mindluster.com/certificate/7863. )

They have over two hundred recorded classes learning from zero to mastery, which I'm sure is gonna help you. They even have a certificate when finish the course. Also the feedbacks from the course are great.

Plus here's the link to the official documentation of laravel: 
(https://laravel.com/docs/10.x)

Have nice studies :)

**Comment by farrasmuttaqin** [comment]
> I need good source to learn laravel

go to https://chat.openai.com/

**Comment by Muhammad-Zain01** [comment]
Learning Laravel, a popular PHP web framework, can be a rewarding experience. Here are some resources to help you get started and master Laravel:

**Official Documentation:**

Laravel has excellent official documentation available at [laravel.com/docs](https://laravel.com/docs). It's well-written, comprehensive, and updated regularly. Start with the documentation to understand the basics and core concepts of Laravel.
**Laracasts:**

Laracasts (laracasts.com) is a subscription-based video tutorial service by Jeffrey Way. It offers a vast collection of high-quality screencasts on Laravel, covering a wide range of topics from beginner to advanced levels. While some content is behind a paywall, the subscription is worth the investment for the depth of knowledge you gain.
**YouTube Tutorials:**

Many developers share free Laravel tutorials on YouTube. Channels like "Traversy Media," "Academind," and "Codecourse" often provide quality Laravel content. Search for Laravel tutorials and filter by the most recent to get up-to-date information.
**Books:**

"Laravel: Up and Running" by Matt Stauffer is a great book for beginners. It covers the fundamentals and provides practical examples.
"Laravel: The Complete Guide" by Josh Lockhart is another book that is highly regarded for its clarity and depth.
**Forums and Community:**

Join the Laravel community on forums like [laracasts.com/discuss](https://laracasts.com/discuss) and [laravel.io/forum](https://laravel.io/forum). Engaging with the community can help you get answers to specific questions and stay updated on best practices.
**Courses on Platforms:**

Platforms like Udemy often have Laravel courses. Look for courses with high ratings and positive reviews. Popular courses include "Laravel 8 from Scratch" by Brad Traversy and "PHP with Laravel for beginners - Become a Master in Laravel" by Piotr Jura.
**Official Laravel Podcast:**

The Laravel community has an official podcast, [Laravel Podcast](https://laravelpodcast.com/), where experienced developers discuss various aspects of Laravel, share tips, and interview key figures in the Laravel ecosystem.
**GitHub Repository:**

Explore the Laravel GitHub repository ([github.com/laravel/laravel](https://github.com/laravel/laravel)) to see the framework's source code. It can help you understand how Laravel works under the hood.
Remember that learning a framework is an ongoing process, and practice is essential. Build small projects, participate in the community, and keep up with the latest updates in the Laravel ecosystem.

**Comment by BhathikaGimhan** [comment]
Learning Laravel can be a rewarding experience, opening doors to building more complex and efficient web applications. Here are some excellent resources to get you started, catering to different learning styles and budgets:

**Free Resources:**

* **Official Laravel Documentation:** The comprehensive and up-to-date documentation is your best friend. It covers everything from basic installation and configuration to advanced concepts like Eloquent and middleware. 

* **Laracasts:** This subscription-based platform offers high-quality video tutorials on various Laravel topics, from beginner to advanced levels. They also have a free series, "Laravel From Scratch," which is a great starting point.

* **Laravel News:** This website publishes regular tutorials, articles, and podcasts on Laravel development. They also have a free email newsletter with Laravel tips and job listings.

* **SitePoint Laravel:** SitePoint offers a variety of free Laravel tutorials and articles, covering various aspects of the framework.

* **YouTube channels:** Many channels on YouTube offer free Laravel tutorials, such as freeCodeCamp, Traversy Media, and Laracasts.

**Paid Resources:**

* **Udemy:** Udemy has a wide range of Laravel courses, from beginner to advanced levels. They often have sales, so you can usually find a good deal.

* **LinkedIn Learning:** LinkedIn Learning offers several Laravel courses, including some taught by Laravel experts.
* **Codecourse:** Codecourse offers in-depth tutorials on various web development frameworks, including Laravel. They have both free and premium content.

* **Books:** Several books cover Laravel development, such as "Laravel Up and Running" by Matt Stauffer and "Laravel: Code Bright" by Jason McCreary.

**Additional Tips:**

* **Start with the basics:** Before diving into Laravel, ensure you have a solid understanding of PHP, HTML, CSS, and JavaScript.
* **Follow a structured learning path:** Don't try to learn everything at once. Choose a learning resource that follows a structured path and stick with it.
* **Practice what you learn:** The best way to learn Laravel is by doing. Try building small projects to put your knowledge into practice.
* **Join the community:** The Laravel community is friendly and supportive. There are many forums, chat rooms, and social media groups where you can ask questions and get help.

**Comment by tamang-manindra** [comment]
Laravel is a popular PHP framework known for its elegant syntax, developer-friendly approach, and robust features. If you're looking to learn Laravel, there are several excellent resources available online to get you started. Here are some recommended sources:

Official Laravel Documentation: The official Laravel documentation is one of the best places to start. It provides comprehensive guides, tutorials, and references for learning Laravel from scratch. You can find it at [Laravel Documentation](https://laravel.com/docs).

Laracasts: [Laracasts](https://laracasts.com/) is a popular online learning platform focused on web development and specifically Laravel. It offers a wide range of video tutorials, courses, and screencasts covering various aspects of Laravel development, from beginner to advanced topics. You can access Laracasts at Laracasts.

YouTube Tutorials: There are numerous YouTube channels dedicated to Laravel tutorials. Channels like Traversy Media, Codecourse, and Academind offer high-quality Laravel tutorials suitable for beginners and intermediate developers.

Books: There are several books available on Laravel, both in print and digital formats. Some recommended titles include "Laravel: Up & Running" by Matt Stauffer and "Learning Laravel" by Nathan Wu.

Community Forums and Blogs: Participating in Laravel community forums such as the Laravel subreddit (r/laravel) or Laravel forums can be a valuable resource for asking questions, sharing knowledge, and staying updated on Laravel news and developments. Additionally, many experienced Laravel developers maintain blogs where they share tutorials, tips, and best practices.

Online Courses: Platforms like Udemy, Coursera, and Udacity offer online courses on Laravel. Look for courses with high ratings and reviews, and check the course curriculum to ensure it covers the topics you're interested in learning.

Contribute to Open Source Projects: Once you have a basic understanding of Laravel, consider contributing to open source projects built with Laravel. This not only helps you gain practical experience but also allows you to learn from experienced developers and contribute to the Laravel community.

Remember that learning Laravel, like any other programming skill, requires practice, patience, and perseverance. Start with the basics, build simple projects, and gradually work your way up to more complex applications. Good luck with your learning journey!
